service/version_bundles.go

```diff
@@ -4,12 +4,20 @@ import (
 	"github.com/giantswarm/versionbundle"
 
 	chartv1 "github.com/giantswarm/chart-operator/service/controller/chart/v1"
+
+	chartconfigv5 "github.com/giantswarm/chart-operator/service/controller/chartconfig/v5"
+	chartconfigv6 "github.com/giantswarm/chart-operator/service/controller/chartconfig/v6"
+	chartconfigv7 "github.com/giantswarm/chart-operator/service/controller/chartconfig/v7"
 )
 
 func NewVersionBundles() []versionbundle.Bundle {
 	var versionBundles []versionbundle.Bundle
 
 	versionBundles = append(versionBundles, chartv1.VersionBundle())
 
+	versionBundles = append(versionBundles, chartconfigv5.VersionBundle())
```

Comment:
Don't we deprecate this `chartconfig` from the latest version?

Reply:
chart-operator is used in the releases with 0.7.0 version. e.g https://github.com/giantswarm/releases/blob/c0dfb19574adf79217e65b6c09df0f33dee1d377/aws.yaml#L115

Without the endpoint these releases don't show up. Getting that fixed and then will see what the long term fix is.